Reverse ETL 2.0 docs (#6571)
segmentio/segment-docs: src/connections/reverse-etl/reverse-etl-source-setup-guides/snowflake-setup.md

   -- role access
   GRANT ROLE segment_reverse_etl TO USER segment_reverse_etl_user;
   ```
7. Add the account information for your source.  
5. Click **Test Connection** to test to see if the connection works.
6. Click **Add source** if the test connection is successful.

> <!--- need to test what information needs to be entered into Segment UI --->

Author: ```suggestion
```

> Learn more about the Snowflake Account ID [here](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html){:target="_blank"}.

Reviewer: ```suggestion
Learn more about the Snowflake Account ID [here](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html){:target="_blank"}.
```
Can we make this link description more accessible?

> Learn more about the Snowflake Account ID [here](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html){:target="_blank"}.

Author: ```suggestion
Learn more about the Snowflake Account ID in Snowflake's [Account identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html){:target="_blank"} documentation.
```
